lib/CMakeLists.txt

```diff
@@ -4,3 +4,5 @@
 # with a Kconfig setting as follows:
 #
 #   add_subdirectory_ifdef(CONFIG_CUSTOM_LIB              custom_lib)
```

Comment:
This comment appears redundant now that the actual line is in the file; can it be removed?